assets/js/selfoss-base.js

```diff
@@ -609,6 +653,19 @@ var selfoss = {
     },
 
 
+    /**
+    * Obtain a localized message for given key.
+    * @param string
+    * @return string
+    */
+    translate: function(key) {
```

Comment:
Why not use `selfoss.ui._()` ?

Reply:
Yeah, that would work too. I just did not see much utility in shortening it since it will still need to be prefixed by `selfoss.ui.` for now.

The function should also accept arguments like `i18nFormat` but I wanted to only implement minimal changes to make this work.

Reply:
I completely forgot that we had `selfoss.ui._`. Especially since so many strings were obtained directly from `#lang`. Changed to that.